protocols/1540-single-cell-discoveries/ngs_prep.ot2.py

```diff
@@ -0,0 +1,179 @@
+from opentrons import labware, instruments, modules, robot
+
+metadata = {
+    'protocolName': 'NGS Prep',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# labware
+sample_plate = labware.load('opentrons-aluminum-block-96-PCR-plate', '2')
+new_plate = labware.load('opentrons-aluminum-block-96-PCR-plate', '3')
+trough = labware.load('trough-12row', '4')
+tips10_rack = [labware.load('tiprack-10ul', slot)
+               for slot in ['5', '6', '7']]
+tips300_rack = [labware.load('tiprack-200ul', slot)
+                for slot in ['8', '9', '10', '11']]
+
+# modules
+magdeck = modules.load('magdeck', '1')
+mag_plate = labware.load('opentrons-aluminum-block-96-PCR-plate',
```

Comment:
use `biorad-hardshell-96-PCR-plate` and you can remove the magdeck engage height